6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,29 @@
+import { createRequire } from "module";
+const require = createRequire(import.meta.url);
+
+//process
+
+const args = process.argv;
+
+function average(num1, num2, num3) {
+  const avg = (num1 + num2 + num3) / 3;
+  console.log(avg);
+}
```

Comment:
What if the input is not a number but a string or something else? I will suggest to make an if statement that checks that the input is number and if not , then it tells the user to use number instead :)